nfts.proto

```diff
@@ -46,6 +46,7 @@ message TransferMintTransaction {
   string sender = 3;
   string recipient = 4;
   string project_id = 5;
+  string transfer_id = 6;
```

Comment:
Nice this is prep for doing a unique transfer record?

Reply:
yup